src/js/workers/hex_parser.js

```diff
@@ -93,13 +93,14 @@ function read_hex_file(data) {
     }
 }
 
+const TIME_LABEL = 'HEX_PARSER - File parsed in';
+
 onmessage = function(event) {
-    const timeParsingStart = microtime(); // track time
+    console.time(TIME_LABEL)
```

Comment:
Sonar issue: missing semicolon.

Reply:
Lol, this will go into eslint for sure 😆

Reply:
Are you going to fix it though? :wink: